pkg/testutils/reduce/reducesql/reducesql.go

```diff
@@ -31,6 +31,7 @@ var SQLPasses = []reduce.Pass{
 	replaceStmt,
 	removeWithCTEs,
 	removeWith,
+	removeCreateDefs,
```

Comment:
For my information, does the order of these matter?

Reply:
The order matters. It's the order in which removals are attempted. So things that have larger impact should be executed earlier which will hopefully decrease the size of the statement as fast as possible, thus decreasing the runtime of the algorithm.